lib/Bio/EnsEMBL/DataCheck/Checks/CompareSchema.pm

```diff
@@ -151,6 +151,9 @@ sub normalise_table_def {
   $table =~ s/[^\)]+\Z//gm;
   $table =~ s/,\s*\)\Z/\n\)/m;
 
+  # Simplify alternative nomenclature for binary columns
+  $table =~ s/CHARACTER SET LATIN1 COLLATE LATIN1_BIN/BINARY/gm;
```

Comment:
@marcoooo - would you prefer this way?
```suggestion
  $table =~ s/BINARY/CHARACTER SET LATIN1 COLLATE LATIN1_BIN/gm;
```

Reply:
@CristiGuijarro  Actually, I am not sure, did you test your DC update on a core DB?

Reply:
I just tested it on a core, it was fine :ok_hand:

Reply:
```
ibsub -m 500Mb run_datachecks.pl -n 'CompareSchema' -host mysql-ens-vertannot-staging -port 4573 -user ensro -dbname adineta_vaga_core_54_107_1
bsub -q standard -M500 -R"select[mem>500] rusage[mem=500]" -n 1  -env all -Is run_datachecks.pl -n CompareSchema -host mysql-ens-vertannot-staging -port 4573 -user ensro -dbname adineta_vaga_core_54_107_1
Job <3687564> is submitted to queue <standard>.
<<Waiting for dispatch ...>>
<<Starting on hl-codon-15-03>>
CompareSchema .. 
# Subtest: CompareSchema
    # Subtest: adineta_vaga, core, adineta_vaga_core_54_107_1, EnsemblMetazoa
        ok 1 - Database schema matches schema defined in file
        1..1
    ok 1 - adineta_vaga, core, adineta_vaga_core_54_107_1, EnsemblMetazoa
    1..1
ok 1 - CompareSchema
1..1
ok
All tests successful.
Files=1, Tests=1,  2 wallclock secs ( 0.07 usr +  0.02 sys =  0.09 CPU)
Result: PASS
```

Reply:
Thank you for that! The one submitted, or the updated one? (so I commit your suggestion or not)

Reply:
The original submission